src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json

```diff
@@ -1,11 +1,11 @@
 {
-    "RightVRSTestVPrefixedBy0_3" : {
-	"expect" : [
-            {
-                "network" : [">=Frontier"],
-                "result" : "invalid"
-            }
-        ],
+    "RightVRSTestVPrefixedBy0_3" :
+    {
+	    "expectException" : 
+        {
+            "<=EIP150": "InvalidVRS",
```

Comment:
oh, I forgot I actully support <= in network translation

Reply:
here it should reject I think not with invalid chain id but with prefixed 00, as the v value limitations. need to check with the v value cap.